pkgs/development/interpreters/python/hooks/python-imports-check-hook.sh

```diff
@@ -1,18 +1,24 @@
+# shellcheck shell=bash
+
 # Setup hook for checking whether Python imports succeed
 echo "Sourcing python-imports-check-hook.sh"
 
 pythonImportsCheckPhase() {
     echo "Executing pythonImportsCheckPhase"
 
-    if [ -n "$pythonImportsCheck" ]; then
-        echo "Check whether the following modules can be imported: $pythonImportsCheck"
-        pythonImportsCheckOutput=$out
-        if [ -n "$python" ]; then
+    if [ -n "${pythonImportsCheck[*]-}" ]; then
+        echo "Check whether the following modules can be imported: ${pythonImportsCheck[*]}"
+        # shellcheck disable=SC2154
+        pythonImportsCheckOutput="$out"
+        if [ -n "${python-}" ]; then
             echo "Using python specific output \$python for imports check"
-            pythonImportsCheckOutput=$python
+            pythonImportsCheckOutput="$python"
```

Comment:
Unnecessary change, right? (But perfectly fine: I just want to check I’m not missing a nuance.)

Reply:
Interesting! I thought that `foo=$bar` didn’t ever need any quoting – or is it just a stylistic lint?

Reply:
It is unnecessary.